src/abstractdatatable/reshape.jl

```diff
@@ -202,21 +202,16 @@ function unstack(dt::AbstractDataTable, rowkey::Int, colkey::Int, value::Int)
     if T <: Nullable
         T = eltype(T)
     end
-    payload = DataTable(Any[NullableArray(T, Nrow) for i in 1:Ncol],
+    payload = DataTable(Any[NullableVector{T}(Nrow) for i in 1:Ncol],
                         map(Symbol, levels(keycol)))
-    nowarning = true
     for k in 1:nrow(dt)
         j = Int(CategoricalArrays.order(keycol.pool)[keycol.refs[k]])
         i = Int(CategoricalArrays.order(refkeycol.pool)[refkeycol.refs[k]])
         if i > 0 && j > 0
-            if nowarning && !isnull(payload[j][i])
-                warn("Duplicate entries in unstack.")
-                nowarning = false
-            end
             payload[j][i]  = valuecol[k]
         end
     end
-    insert!(payload, 1, NullableArray(levels(refkeycol)), _names(dt)[rowkey])
+    denullify!(insert!(payload, 1, levels(refkeycol), _names(dt)[rowkey]))
```

Comment:
Unstack needs to use NullableArrays for cases where not all columns are completely filled with values. In the case of incompletely full Vectors, Vector{String}s will have #undef values which break lots of things immediately. I thought the best approach would be to generate a DataTable of NullableArray as the current code does and then simplify the DataTable before returning it.